### PR TITLE
add support for sorting based on presence in file

### PR DESCRIPTION
**Pull request by rbren**
This adds a command-line flag, `--sort`, which allows the user to sort either alphabetically, or based on the ordering of values in the values.yaml file.

I've defaulted to `--sort=file` here, as I think that would be the more desired behavior. But I'm happy to revert to using `alpha` as the default.

### CODE REVIEW

File: cmd/helm-docs/command_line.go [hunk 1]
```diff
@@ -48,6 +48,7 @@ func newHelmDocsCommand(run func(cmd *cobra.Command, args []string)) (*cobra.Com
 	command.PersistentFlags().StringP("log-level", "l", "info", logLevelUsage)
 	command.PersistentFlags().StringP("output-file", "o", "README.md", "markdown file path relative to each chart directory to which rendered documentation will be written")
 	command.PersistentFlags().StringP("template-file", "t", "README.md.gotmpl", "gotemplate file path relative to each chart directory from which documentation will be generated")
+	command.PersistentFlags().StringP("sort", "s", "file", "order in which to print the values (alpha or file)")
```
Comment (norwoodj) [inline cmd/helm-docs/command_line.go:51]: This looks great, but please change the name of this flag to `sort-values-order` and make the default `alpha`. Don't want to change default behaviour. Thanks!

**Review by norwoodj — CHANGES_REQUESTED** [review]

File: cmd/helm-docs/command_line.go (hunk 1 above)
Comment (rbren) [reply cmd/helm-docs/command_line.go]: 👍  done!

**Comment by norwoodj** [comment]
@rbren I've used your commits and made some slight changes on top of them, as well as rebasing. Your commits will appear in release v1.3.0, coming out later tonight

**Comment by rbren** [comment]
Awesome, thanks a bunch! Looking forward to checking out 1.3

**Comment by jasondamour** [comment]
I don't think is documented anywhere, and I had to stumble across this PR. Great feature, but needs docs! @norwoodj

**Pull request closed without merging**